common/src/python/centers/center_group.py

```diff
@@ -522,6 +549,71 @@ def __add_user_roles_to_project(self, *, user: User, project_id: str,
 
         return project.add_user_roles(user=user, roles=roles)
 
+    def __add_user_to_redcap_project(
+            self, *, user: User,
+            form_ingest_project: 'FormIngestProjectMetadata',
+            authorizations: Authorizations, parameter_store: ParameterStore,
+            redcap_path: str) -> bool:
+        """Adds user to the respective REDCap project for direct data entry.
+
+        Args:
+          user: the user to add
+          form_ingest_project: metadata about form ingest project
+          authorizations: the authorizations for the user
+          parameter_store: AWS parameter store connection
+          redcap_path: Parameter path prefix for REDCap API credentials
+
+        Returns:
+          True if user was added, False if errors occurred
+        """
+        assert user.id, "requires user has ID"
+
+        if not form_ingest_project.redcap_projects:
+            log.warning('REDCap project metadata not available for %s',
+                        form_ingest_project.project_label)
+            return False
+
+        activities = authorizations.get_activities()
+
+        success = True
+        for redcap_prj in form_ingest_project.redcap_projects.values():
+            if (redcap_prj.label.upper() == ENROLLMENT_MODULE
+                    and 'submit-enrollment' in activities):
+                has_permissions = True
+            # assume all other modules are covered by submit-form permission
+            elif 'submit-form' in activities:
+                has_permissions = True
+            else:
+                has_permissions = False
+
+            if not has_permissions:  # User doesn't have submission privileges
```

Comment:
Can you rewrite this to avoid using the `has_permissions` flag? also, avoiding using else